### DISCUSSION TITLE
Will metaspades.py or rnaspades.py will provide redundant contigs?

### DISCUSSION THREAD

**Discussion by SarahIsme** [ablab/spades#886] [Q&A] [answered]
Thanks for this great tool!

When I use this tool to assemble illumina reads (150nt), I am curious whether redundant contigs will be generated？

Because lots of redundant reads are in my fastq files.

Thanks in advance!

Sarah

**Comment by asl** [comment, marked as answer]
By "redundant" did you mean contigs with identical sequences? Certainly not, assemblers work in other way

**Reply by SarahIsme** [reply]
Thanks for your reply！

Yes，the “redundant” in my question means the contigs with identical sequences.
